src/site/stages/build/process-cms-exports/filters.js

```diff
@@ -1,14 +1,41 @@
 const path = require('path');
-const { getContentModelType, getAllImportsFrom } = require('./helpers');
+const {
+  getContentModelType,
+  getAllImportsFrom,
+  typeProperties,
+} = require('./helpers');
+const { omit } = require('lodash/fp');
 
 // Dynamically read in all the filters
 // They must be named after the content model type (E.g. node-page.js)
 const filtersDir = path.join(__dirname, 'transformers');
 const filters = getAllImportsFrom(filtersDir, 'filter');
 
 /**
- * When reading through entity properties, ignore these.
+ * A list of properties to ignore.
+ *
+ * This list comes from the typeProperties, which we never want to
+ * expand, and a temporary list of properties we don't want to filter
+ * out on a per-content-model basis.
+ *
+ * Additionally, this is useful for temporarily ignoring entity
+ * expansion of certain properties before we've created a filter for
+ * that content model.
  */
+const ignoreList = typeProperties.concat([
```

Comment:
`typeProperties` is an array of property names which describe the type of entity it is. Usually this is `type`, but can also be `bundle` or `vid` (if memory serves).

We're building this `ignoreList` using `typeProperties` as a base because an entity's `type` _looks_ like an entity reference (it has a `target_uuid` and `target_type`), but will never be found in the tome-sync output, so if we try expanding them, it'll fail.